src/python/pants/backend/python/tasks/build_local_python_distributions.py

```diff
@@ -103,6 +103,9 @@ def _copy_sources(self, dist_tgt, dist_target_dir):
                                   src_relative_to_target_base)
       shutil.copyfile(abs_src_path, src_rel_to_results_dir)
 
+  def _request_single(self, product, subject):
+    return self.context._scheduler.product_request(product, [subject])[0]
```

Comment:
This technically is not supposed to be exposed yet... should leave a comment here pointing to https://github.com/pantsbuild/pants/issues/5440.

Reply:
Left a comment pointing to #4769 after offline discussion in 241692f, let me know if that's what you were thinking.